tck/tck-dist/pom.xml

```diff
@@ -0,0 +1,141 @@
+         <version>${project.version}</version>
+         <type>pom</type>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-tck-common</artifactId>
+         <version>${project.version}</version>
+         <classifier>sources</classifier>
+         <optional>true</optional>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-tck-tests</artifactId>
+         <version>${project.version}</version>
+         <type>jar</type>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-tck-tests</artifactId>
+         <version>${project.version}</version>
+         <type>pom</type>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-tck-tests</artifactId>
+         <version>${project.version}</version>
+         <classifier>sources</classifier>
+         <optional>true</optional>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-tck-tests-plugability</artifactId>
+         <version>${project.version}</version>
+         <type>jar</type>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-tck-tests-plugability</artifactId>
+         <version>${project.version}</version>
+         <type>pom</type>
+      </dependency>
+      <dependency>
+         <groupId>jakarta.json</groupId>
+         <artifactId>jakarta.json-tck-tests-plugability</artifactId>
+         <version>${project.version}</version>
+         <classifier>sources</classifier>
+         <optional>true</optional>
+      </dependency>
+   </dependencies>
+
+   <profiles>
+      <profile>
+       <id>EFTL</id>
+         <properties>
+            <license>EFTL</license>
+            <bundle-name>jakarta-jsonp-tck</bundle-name>
+         </properties>
+      </profile>
+      <profile>
+         <id>EPL</id>
+         <properties>
+            <license>EPL</license>
+            <bundle-name>jsonp-tck</bundle-name>
+         </properties>
+         <activation>
+            <activeByDefault>true</activeByDefault>
+        </activation>
+      </profile>
+   </profiles>
+
+   <build>
+      <plugins>
+         <plugin>
+            <artifactId>maven-assembly-plugin</artifactId>
+            <executions>
+               <execution>
+                  <id>distribution</id>
+                  <phase>package</phase>
+                  <goals>
+                     <goal>single</goal>
+                  </goals>
+                  <configuration>
+                     <descriptors>
+                        <descriptor>src/main/assembly/assembly.xml</descriptor>
+                     </descriptors>
+                     <finalName>${bundle-name}-${project.version}</finalName>
```

Comment:
remove finalName, it's not needed as maven will still deploy the artifact based on the artifactId

Reply:
`<finalName>${bundle-name}-${project.version}</finalName>` will be use for naming generated TCK zip bundle(for EPL license distribution - jsonp-tck-2.1.0-SNAPSHOT.zip, EFTL license distrubution- jakarta-jsonp-tck-2.1.0-SNAPSHOT.zip)